lib/rex/json_hash_file.rb

```diff
@@ -85,7 +85,7 @@ def parse_data(data)
     begin
       JSON.parse(data)
     rescue JSON::ParserError => e
-      # elog("JSONHashFile @ #{path} was corrupt: #{e.class} #{e}"
+      elog("JSONHashFile at #{path} was corrupt", error:e)
```

Comment:
Is there any context on why this was commented in the first place?

Reply:
None. Commit logs don't mention anything, and the call doesn't seem to affect anything negatively. I assume it's a change made during local debugging that got through PR review